programs/lz4io.h

```diff
@@ -116,6 +125,10 @@ int LZ4IO_setSparseFile(LZ4IO_prefs_t* const prefs, int enable);
 /* Default setting : 0 == no content size present in frame header */
 int LZ4IO_setContentSize(LZ4IO_prefs_t* const prefs, int enable);
 
+int LZ4IO_getCompressedFilesInfo(const char** inFileNames,const size_t ifnIdx);
```

Comment:
This is the only function that is being called. 
So maybe it's the only one that deserves to be exposed.

Keep the rest internal.
There is a universal rule of API design that the less capabilities are exposed, the more implementation flexibility is preserved.

Also : naming: prefer `display` to differentiate from `get`, since in this case, information is not actually extracted, it's purely pushed onto the screen.